data-service/src/main/resources/application.yaml

```diff
@@ -49,5 +49,11 @@ feign:
         loggerLevel: basic
     default-config: default
 
+# Default development & testing static config, not suitable for production
+web:
+  client:
+    audit-service: http://localhost:8081
```

Comment:
potential port change to avoid port conflicts

Reply:
done